domains/Discussions/Database/Migrations/2020_10_12_104113_create_questions_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Domains\Accounts\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateQuestionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
Nit: the dockblocks are unnecessary because they don't give any more useful information.

Also, we should add the return type hints.